### ISSUE TITLE
Wrong command in _deploy (service.py)

### ISSUE THREAD

**Issue by elmalto** [rgardler/acs-cli#37]
I think the lines 

```
 def _deploy(self, name):
    command = "azure group deployment create"
    command = command + " " + self.config.get('ACS', 'dnsPrefix')
    command = command + " " + name
    command = command + " --template-uri " + self.config.get('Template', 'templateUrl')
    command = command + " -p '" + json.dumps(self.config.getACSParams()) + "'"
    os.system(command)
```

should be more like

```
 def _deploy(self):
    command = "azure group deployment create"
    command = command + " " + self.config.get('ACS', 'name')
    command = command + " " + self.config.get('ACS', 'dnsPrefix')
    command = command + " --template-uri " + self.config.get('Template', 'templateUrl')
    command = command + " -p '" + json.dumps(self.config.getACSParams()) + "'"
    os.system(command)
```

or am I missing something here? in e.g. create you pass the dnsPrefix and then get an error that the resource group `dns prefix` does not exist ?

**Comment by rgardler** [comment]
You are correct. I always use the same value for resource group and DNS prefix so never noticed this. Fixing now - thanks for debugging

**Comment by rgardler** [comment]
Actually your fix is not quite right. We used to default to the dnsName being the same as the resourceGroup name. This was subsequently made configurable but it appears we neglected to update this code. 

The 'name' here is the deployment name not the name of the resource group. It is passed in as a parameter. I've clarified the name parameter and thus the code should be:

  def _deploy(self, deploymentName):
    command = "azure group deployment create"
    command = command + " " + self.config.get('Group', 'name')
    command = command + " " + deploymentName
    command = command + " --template-uri " + self.config.get('Template', 'templateUrl')
    command = command + " -p '" + json.dumps(self.config.getACSParams()) + "'"

I've modified the tests to cover this case and will commit the change once I've confirmed they pass and thus this change is correct.
